reactor-core/src/test/java/reactor/core/publisher/LambdaMonoSubscriberTest.java

```diff
@@ -153,7 +153,7 @@ public void noSubscriptionConsumerTriggersRequestOfMax() {
 	}
 
 	@Test
-	public void onNextConsumerExceptionBubblesUpDoesntTriggerCancellation() {
+	public void onNextConsumerExceptionHandledByErrorHandler() {
```

Comment:
I would keep at least one additional test where the `errorHolder::set` is omitted, to demonstrate that if there is no error consumer, the exception in nextConsumer bubbles up wrapped in an `ErrorCallbackNotImplemented` exception.